src/jobs_auto_report/action.py

```diff
@@ -0,0 +1,106 @@
+from opensearchpy import OpenSearch
```

Comment:
I prefer moving everything from ``src/jobs_auth_report`` to be under ``src/prowjobsscraper``
The reason for that it that it is usually surprising (in a bad way) users of the prow-jobs-scraper module to find two ways to import stuff from the same code-base (both ``from prowjobsscraper import X`` and ``from jobs_auto_report import Y``)

Reply:
I think we should eventually rename this project which is composed of a scraper and a reporter (I named it `prow-insights` in app-sre)

Reply:
@danmanor can you do this? ^

Reply:
let's tackle this with another PR
@danmanor let's add this task on the list :)